Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/Internationalization.rst

```diff
@@ -38,8 +38,8 @@ use the i18n service API to obtain these verified ``Locale`` objects.
   the ``Neos.Flow.i18n.scan.includePaths`` setting. This is useful to restrict the scanning
   to specific paths when you have a big file structure in your package ``Resources``.
   You can also blacklist folders through ``Neos.Flow.i18n.scan.excludePatterns``.
-  By default all folders except 'node_modules', 'bower_components' and any folder starting
-  with a dot will be scanned.
+  By default the ``Public`` and ``Private/Translations`` folders, except 'node_modules',
```

Comment:
The node_modules and bower_components folders are normally subfolders of the Private folder, which is nor scanned anymore, so excludePatterns and the hint could be removed also?

Reply:
Sometimes people seem to do have node_modules and other such folders in `Resources\Public`, at least according to Aske as of https://github.com/neos/flow-development-collection/pull/385#issuecomment-226050606